tests/functional/relation_tests/dynamic_table_tests/models.py

```diff
@@ -28,6 +28,20 @@
 """
 
 
+DYNAMIC_ICEBERG_TABLE = """
+{{ config(
+    materialized='dynamic_table',
+    snowflake_warehouse='DBT_TESTING',
+    target_lag='2        minutes',
```

Comment:
tab where there should be a space by chance? :)

Reply:
This might be outdated since we've confirmed how this works, but the issue I'm implicitly testing is that Snowflake parses this string and stores it in their metadata as `2 minutes`. Honestly, we probably should just say that this is not documented support, hence we are not testing it.